extensions/interactions/MultipleChoiceInput/directives/oppia-interactive-multiple-choice-input.component.ts

```diff
@@ -117,6 +128,27 @@ export class InteractiveMultipleChoiceInputComponent implements OnInit {
       this.orderOfChoices.push(this.choices[i].originalIndex);
     }
 
+    // Setup voiceover.
+    this.displayedCard = this.playerTranscriptService.getCard(
+      this.playerPositionService.getDisplayedCardIndex());
+    this.recordedVoiceovers = this.displayedCard.getRecordedVoiceovers();
+
+    // Combine labels for voiceover.
+    let combinedChoceLabels = '';
+    for (const choice of choices.value) {
+      // If the labels are in html format, remove the tags and leave the
+      // content only.
+      const cleanChoiceLabel = choice.html.replace(/<[^>]+>/g, '');
+
+      combinedChoceLabels += cleanChoiceLabel + '. ';
+    }
```

Comment:
This is also defined in the component above, can we define it in one place and then use it here and above?

Reply:
Good idea, to avoid duplication. Wondering if I should refactor it out as a helper function in state-card.model.ts, recorded-voiceovers.model.ts or audio-translation-manager.service.ts. @vojtechjelinek What do you think?

Reply:
done, thanks!